get-started/mobile-traffic/configure-android.md

```diff
@@ -9,55 +9,54 @@ position: 10
 previous_url: /knowledge-base/configure-android
 ---
 
-#### Environment
+# How to Capture and Inspect Android Traffic
```

Comment:
This is no longer a KB article so you can use a general structure that fits getting started articles. Such articles benefit more from descriptive headings.